impl/src/rendering/templates/partials/features.template

```diff
@@ -0,0 +1,21 @@
+
+    {%- if crate.features.features | length != 0 or crate.features.targeted_features | length != 0 %}
+    crate_features = [
+    {%- for feature in crate.features.features %}
+        "{{feature}}",
+    {%- endfor %}
+    ]{% for targeted_feature in crate.features.targeted_features %} + selects.with_or({
+        (
+    {%- for platform in targeted_feature.platforms %}
+            "@rules_rust//rust/platform:{{platform}}",
+    {%- endfor %}
+        ): [
+    {%- for feature in targeted_feature.features %}
+            "{{ feature }}",
+    {%- endfor %}
+        ],
+        "//conditions:default": [],
+    })
+    {%- endfor -%},
+    {%- endif %}
+    
```

Comment:
Is this extra line causing that new line to be added to all of the build files?

Reply:
With my last patch, the generated .bazel files match the buildifier versions for me. I was missing a subtlety of Tera.